test/e2e/operator.go

```diff
@@ -124,6 +124,9 @@ var _ = Describe("ARO Operator - Internet checking", func() {
 
 var _ = Describe("ARO Operator - Geneva Logging", func() {
 	It("must be repaired if DaemonSet deleted", func(ctx context.Context) {
+		if _env.IsLocalDevelopmentMode() {
```

Comment:
Purely curiosity and not a request for change, but why did you find that you had to change this during your work? Why didn't we have this test disabled for local dev mode previously?

Reply:
MDSD pods require a certificate signed by OneCert to connect to Geneva.  We were able to use OneCert in the MSIT tenant but it is not available in the new tenant.